File: other/CS50/w6/class/phonebook.py
import csv

with open("phonebook.csv", "a") as file:
    # that's for automatic closing (instead of file.open and file.close)

    name = input("Name: ")
    number = input("Number: ")

    writer = csv.DictWriter(file, fieldnames=["name", "number"])
    writer.writerow({"name": name, "number": number})

File: other/CS50/w4/class/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    FILE *file = fopen("phonebook.csv", "a");

    string name = get_string("name: ");
    string number = get_string("number: ");

    fprintf(file, "%s, %s\n", name, number);

    fclose(file);
}